en/core-libraries/httpclient.rst

```diff
@@ -428,6 +396,63 @@ XML data is decoded into a ``SimpleXMLElement`` tree::
 The decoded response data is stored in the response object, so accessing it
 multiple times has no additional cost.
 
+Accessing Response Headers
+--------------------------
+
+You can access headers through a few different methods. Header names are always
+treated as case-insensitive values when accessing them through methods::
+
+    // Get all the headers with array values.
```

Comment:
How about "Get all the headers as associative array"?

>      * @return string[][] Returns an associative array of the message's headers.
     *     Each key MUST be a header name, and each value MUST be an array of
     *     strings for that header.

http://www.php-fig.org/psr/psr-7/#3-3-psr-http-message-responseinterface